### DISCUSSION TITLE
List of Items I intend to add, as well as static objects

### DISCUSSION THREAD

**Discussion by DeivenGreer** [cataclysmbn/Cataclysm-BN#4308] [General]
Some items may already be added to the game. It's too much to check, so I've compiled every thought into this. I would rather have an imperfect list than no list and I hope that this work will be helpful to the development of the game and the community at large.

### **Food Preservation Techniques:**

_Vessel Required:_
Canning
Pickling
Brining
Force/Forcemeat
Aspic/Jellying
Juicing/Pulping

_Equipment Required:_
Cooking
Freeze Drying
Drying/Jerkying
Dehydrating
Smoking
Powdering
Blanching
Curing/Candying
Drawing/Teas
Rehydrating

### **Non-Perishable Edibles:**

_Dried/Dehydrated:_
Dried Beans
Dried Mushrooms
Dried Hallucin. Mushrooms
Dehydrated Vegetable
Dehydrated Fruit
Dehydrated Meat
Dehydrated Meat- Human
Dehydrated Offal
Dehydrated Flesh
Dehydrated Fish
Dehydrated Insect
Protein Powder
Soylent Powder
Powdered Milk
Powdered Coconut
Powdered Egg
Powdered Meat/Pemmican
Powdered Coffee Beans
Powdered Alchohol
Flour- Acorn
Flour- Grain
Flour- Nut
Flour- Rice
Flour- Starch
Flour- Tree Bark
Processed Starch
Dried Salad
Freeze-dried Beans
Freeze-dried Mushrooms
Freeze-dried Hallucin. Mushrooms
Freeze-dried Vegetable
Freeze-dried Fruit
Freeze-dried Meat
Freeze-dried Meat- Human
Freeze-dried Offal
Freeze-dried Flesh
Freeze-dried Fish
Freeze-dried Insect
Freeze-dried Milk
Freeze-dried Egg
Freeze-dried Meat/Pemmican
Freeze-dried Coffee Beans

_Ingredients:_

Bonemeal/Bone
Bouillon Cubes
Coconut Oil
Cornstarch
Dark Chocolate
Dried Herbs
Dried Spices
Energy Drinks
Extra Virgin Olive Oil
Fine Wine
Hard Liquor
Honey
Honey- Candied
Honey- Forest
Lard
Maple Syrup
Marmite
Molasses
Pemmican
Purified Water
Saline Solution
Salt
Salt- Block, 5 lbs
Salt- Block, 25 lbs
Salt- Seasoned
Seeds
Soft Drinks
Soy Sauce
Sports Drinks
Sugar- Brown
Sugar- Imitation
Sugar- White
Tallow
Vanilla Extract
Vegemite
Vinegar- Apple Cider
Vinegar- Distilled
Vinegar- Fruit
Yeasts (Lasts 2 years, edge case.)

_Canned Goods:_

Asparagus Cuts & Tips
Aspic
Beef Stew
Brandied Oysters
Brandied Peaches
Canned Ham
Canned Honey
Chicken Alfredo
Chicken Breast
Chicken Stew
Chicken- White & Dark
Chili Cheese
Corned Beef Hash
Dark Sweet Cherries
Green Beans
Ham & Potatoes
Meatloaf & Mashed Taters
Mixed Fruit
Mixed Vegetables
MRES
No Bean Chili
Pickled- Egg
Pickled- Fruit
Pickled- Meat
Pickled- Offal
Pickled- Veggie
Pigs Feet
Pineapple Juice
Pink Grapefruit
Plum Tomatoes
Pork Hocks
Roast Beef & Gravy
Sauerkraut
Soups
SPAM (Could be some other name for copyright, idk)
SPAM- Bacon
SPAM- Black Pepper
SPAM- Chorizo
SPAM- Garlic
SPAM- Hickory Smoke
SPAM- Hot & Spicy
SPAM- Jalapeno
SPAM- Lite
SPAM- Portuguese Sausage
SPAM- Spread
SPAM- Teriyaki
SPAM- Tocino
SPAM- Turkey
Stroganoff
Tamales
Turkey Breast
Vienna Sausages

_Other:_

Candy
Coffee
Crackers
Fruit Leather
Fruitcake
Ghee
Hard Cheese in Wax
Hardtack
Herbal Tea
Irradiated Foods
Ramen Noodles
Tea
Toaster Pastries

_Spices:_

Salt (already mentioned)
Pepper
Bay Leaf
Ginger
Mustard
Italian Seasoning
Five Spice
Tumeric
Cinnamon
Curry Powder
Chili Powder
Garlic

_Candies:_

Caramel
Candy Cigarrettes
Chewing Gum
Chewing Gum- Caffinated
Chewy Candies
Chocolate Coffee Squares
Chocolate Bars
Chocolate Pretzels
Cough Drops/Lozenges
Lollipop
Malted Milk Balls
Maple Syrup Candies
Marshmellows
Peanut Butter Candies
Peppermint Patties
Powdered Sticks
Taffy
Toffee
 

_Junk Food: (Perishable?)_
Beer Snack- Caramel Corn
Beer Snack- Nuts/Beer Nuts
Candied Meats
Cereal
Cereal- Sugar
Cereal- Corn
Cereal- Wheat
Chocolate Wafers
Cookies
Cookie Jelly
Cupcake Mix
Doughnuts
Graham Crackers
Modjeska
Popcorn
Pork Sticks
Potato Chips
Pretzels
Sweet Sausage
TV Dinner

_MRES:_

-C-Ration
-Iron Ration
-Reserve Ration
-Jungle Ration
-Mountain Ration
-LRRP Ration
Pork Patty
Ham & Chicken Loaf
Beef Patty (Hockey Puck)
Beef Slices in BBQ
Beef Stew
Frankfurters w/ Beans
Turkey diced w/ Gravy
Beef diced w/ Gravy
Chicken a la King
Meatballs &BBQ
Ham Slices
Ground Beef w/ Spiced Sauc
Chicken Loaf
Pork w/ Rice in BBQ
Corned Beef Hash
Chicken Stew
Omelet w/ Ham
Spaghetti w/ Meat Sauce
Meatballs w/ Tomato
Tuna w/ Noodles
Chicken w/Rice
Escalloped Potatoes w/ Ham
Smokey Franks
Pork Chow Mein
Chili w/ Macaroni
Chicken Stew
Grilled Chicken
Pasta w/ Vegetables
Cheese Tortellini
Chicken Parmesan
Boneless Pork Chop
Beef Ravioli
Turkey Breast w/ Gravy
Beef w/ Mushrooms
Thai Chicken
Chicken w/ Cavatelli
Bean & Rice Burrito
Beef Teriyaki
Chicken w/ Salsa
Meat Loaf w/ Gravy
Pasta w/ Alfredo Sauce
Boneless Pork Jamaican
Grilled Beefsteak
Country Captain Chicken
Chicken Tetrazzini
Chicken w/ Thai Sauce
Beef Enchilada
Jambalaya
Pork Rib
Black Bean & Rice Burrito
Manicotti w/ Vegetables
Roast Beef
Veggie Burger
Cajun Rice, Beans, & Sausage
Chicken Fajitas
Sloppy Joe Filling
Spicy Penne Pasta
Tuna in Pouch
Vegetable Lasagna
Chicken Pesto & Pasta
Maple Sausage
Beef Brisket
Veggie Burger w/ BBQ
Buffalo Chicken
Mediterranean Chicken
Southwest Beef
Ratatouille
Lemon Pepper Tuna
Chicken, Tomato, Fea
JalapenoPepper
Jack Beef Patty
Asian Beef Strips
Shredded BBQ Beef
Diced Chicken
Beef Taco
Vegetarian Taco Pasta
Mex Style Chicken Stew
-MRK (Kosher for Passover)
-Humanitarian Daily Ration
-Meal, Cold Weather (MCW)
-Meal, Alt Regionally Custom
-Meal, Long Range Patrol
-FPSGPI (Gen Purpose Impr)
-Unitized Group Ration
_French RCIR:_
Beef Salad
Salmon Rice Veg
Stewed Beef
Saute of Rabbit
Paella
Mutton Stew Flagolets
Stewed Lamb
Coalfish & Noodles
Earthenware Cassoulet
Pork & Greens
Stewed Pork
Pork Salad
Chicken & Greens Parisienne
Sausage & Lentils
_FSR (French Survival Ration):_
Tuna Fish Pate
Salmon Pate
Potted Meat of Mackerel
Salt Beef in Mustard
Tuna in Sauce
Sardine Medallions
Marined Mackerels
Duck Pate
Liver Pate (Pork)
Potted Meat of Pork
Pork Pate & Mushrooms
Ham Pate
Pork Pate
Tuna Catalane
_German EPA:_
Lentil Stew w/ Sausage
Tagliatelle Bolognese
Ćevapcicci (Yugoslav)
Goulash w/ Potatoes
Veg Stir Fry w/ Tofu
Beef Patties w/ Tom Sauce
Powidl (Germ/Czh Jam)
Russian IRP
Tushonka (Rus Stewed Meat)
Salo (Rus Pig Fatback)
_Italian RVSC:_
Module A- Yellow
Module B- Red
Module C- Gray
Module D- Green
Module E- White
Module F- Pink
Module G- Blue
_British 24 Hr Ration Pack
British 4 Man Ration Pack
British 10 Man Ration Pack_
-Civilian Emercency Food Rat
-Survival Bars
-BP-5 Compact Food
_**American Space Meal
Russian Space Meal
Chinese Space Meal**_

Soups:

Beef Soup
Broth- Beef
Broth- Bone
Broth- Chicken
Broth- Fish
Broth- Vegetables
Butternut Squash Bisque
Cheddar Cheese Soup
Chicken Alfredo Soup
Chicken Noodle Soup
Chicken Pasta Soup
Chicken & Rice Soup
Chicken Tortilla Soup
Chicken & Dumplings
Clam Chowder- Chunk
Clam Chowder- Cream
Corn Chowder
Crab Soup
Cream of Chicken
Cream of Mushroom
Cream of Potato
Gouda Bisque
Minestrone Soup
Onion Soup
Pasta Fagioli Soup
Pea Soup
Potato Soup
Potato Broccoli Cheese Soup
Pot Pie Chicken Soup
Ramen- Canned
Spaghetti O's
Spinach & Meatballs
Steak & Cheese Soup
Stock- Bone
Stock- Beef
Stock- Coconut
Stock- Cream Base Powder
Stock- Ginger
Stock- Hondashi
Stock- Mushroom
Stock- Pumpkin
Stock- Scallop
Stock- Seafood
Stock- Vegetable
Tomato Soup
Vegetable Medley Soup
White Chicken Chili Soup

Now we get into the perishables. That's right, all the stuff above is just that which won't spoil when packaged correctly. 

### **Perishable Edibles:**
_Grains:_
Barley
Buckwheat
Cattail
Corn
Farro
Hops
Millet
Oats
Quinoa
Rapeseed
Rice- Brown
Rice- Wild
Rye
Spelt
Sorghum
Wheat
Wild Grass
Teff
 
Grain Products:
Atole
Biscuit
Bread
Bread- Cornbread
Bread- Flatbread
Cakes
Canola Oil
Chips
Congee (Rice Gruel)
Cookies
Couscous
Crackers
Crepes
Dough
Granola
Grits
Grog
Gruel
Hominy
Johnnycake
Masa- Corn Dough
Muesli
Mush
Pies
Porridge
Rice Pudding
Starch
Tortilla
Wastebread
 
_Pasta:_
Spaghetti
Lasagne
Macaroni
 
 
_Fruits: (&Rotten Varieties)_ (I plan on adding far more to this.)
Apple
Apricot
Banana
Bael
Breadfruit
Berry- Black
Berry- Blue
Berry- Cran
Berry- Elder
Berry- Lingon
Berry- Mulberry
Berry- Raspberry
Berry- Strawberry
Cantaloupe
Cherry
Citron
Coconut
Fig
Grape
Honeydew
Jackfruit
Kiwi
Lemon
Lime
Mango
Mangosteen
Morinda
Muskmelon
Orange
Pawpaw
Papaya
Passionfruit
Peach
Pear
Persimmon
Pineapple
Plum
Pomegranate
Pomelo
Pineapple
Pumpkin
Soursop
Vanilla
Watermelon
 
 
_Vegetables: (Normal & Jarred)_
Alfalfa
Amaranth
Artichoke
Asparagus
Avocado
Bambara Groundnut
Beans- Mixed
Beans- Garbanzo
Beans- Green
Beets
Broccoli
Cabbage
Carrot
Cattail
Celery
Chickpea
Cucumber
Eggplant
Kaniwa
Leek
Lettuce
Onion
Mushrooms
Peas
Pepper- Bell
Pepper- Jalapeno
Potato
Potato- Sweet
Radish
Rhubarb
Spinach
ZUCChini
Sugar Beets
Taro
Tofu
Tomatillo
Turnip
Yams
 
_Flowers/Wild/Forage:_
Acorns
Chamomile
Chestnuts
Crysanthemum
Cumin
Fennel
Ginger
Hazelnut
Jerusalem Artichoke
Kelp
Lovage
Mint
Mugwort
Mushroom- Black
Mushroom- Blue
Mushroom- Brown
Mushroom- Red
Mushroom- Yellow
Mushroom- Morel
Mushroom- Fly Amanita
Nettle
Nutmeg
Oleander
Oregano
Olives
Paprika
Parsley
Pea Pods
Pine Nut
Rosemary
Sage
Sassafrass
Snapdragon
Sugar Beets
Thyme
Tumeric
Wild Grasses
Woad
 
_Nuts:_
Almond
Walnut
Pistachio
Peanut
Pecan
Macadamia
 
_Meat:_
Meat-Red
Meat- Light Bird (Chicken/Small Game)
Meat- Dark Bird (Turkey/Large Game)
Meat- Frog
Meat- Reptile
Meat- Fish Fillet
Meat- Mutton
Meat- Patty
Meat- Pork
Meat- Rabbit
Meat- Rodent
Meat- Steak
Cracklins
Offal
Salted Meat
Salted Fish
Bacon
Bacon Bits
Bacon Rashers
Bologna
Hotdog
Eggs- Bird
Eggs- Insect
Eggs- Reptile
 
_Condiments:_
Aioli
Ajvar
Anchovy Paste
Barbecue Sauce
Chili Oil
Chili Sauce
Chutney
Cocktail Sauce
Compote (French Fruit Paste)
Cooking Oil
Cream Cheese
Crushed Red Pepper
Fish Paste
Fish Sauce
Fruit Preserves
Fry Sauce
Guacamole
Honey
Honey Dill
Horseradish
Hot Sauce
Jalapenos
Jam- Fruit
Karo (Corn Syrup)
Ketchup
Ketchup- Banana
Ketchup- Curry
Ketchup- Mushroom
Ketchup- Fruit
Maple Syrup
Mayo
Mignonette Sauce
Miracle Whip
Miso
Mumbo Sauce
Mustard
Mustard- Dijon
Mustard- Oil
Nutella
Olive Oil
Pepper Jelly
Pesto
Pico de Gallo
Relish
Remoulade
Salad Dressing- Blue Cheese
Salad Dressing- Caesar
Salad Dressing- French
Salad Dressing- Ginger
Salad Dressing- Honey Dijon
Salad Dressing- Hummus
Salad Dressing- Italian
Salad Dressing- Ranch
Salad Dressing- Russian
Salad Dressing- Tahini
Salad Dressing- Thousand Island
Salad Dressing- Vinaigrette
Salad Dressing- Wafu Dressing
Salsa
Sauerkraut
Sesame Oil
Soy Sauce
Steak Sauce
Syrup
Tartar Sauce
Teriyaki Sauce
Vinegar
Vinegar- Black

**Medical Consumables:**

_Liquids:_
Drowsy Cough Syrup
Non-Drowsy Cough Syrup
Antacid
 
_Items:_
Adrenaline Shot
Bandage
Bleach
Contact Lenses
Contact Lens Solution
Cotton Balls
Denatured Alch
Ethanol
First Aid Kit
Flu Shot
Hydrogen Peroxide
Inhaler
Medical Gauze
Medical Tape
Multivitamin
Multivitamin Gummy
Oxygen Pack
Oxygen Tank/Mask
Pink Eye Solution
Poppy Bud
Rag
Rag- Dirty
Stim Shot
Syringe
Tourniquet
Vicks Jet Injector
Water Purification Tablet
 
_Pills:_
Caffeine
Antibiotic
Antifungal
Antiparasitic
Calcium Tablet
Potassium Iodide
Prussian Blue
 
_Antipsychotic:_
Abilify
LSD
Seroquil
Thorazine
Zyprexa
 
_Sedative/Analgesic:_
Asprin
Ambien
Codeine
Morphine
Oxycodone
 
_Stimulants:_
Adderall
Concerta
Crack
Crack Pipe
Cocaine
Molly
Ritalin
Strattera
Vyvase
 
_Antidepressants:_
Oleptro
Paxil
Prozac/Fluoxetine
Zoloft
 
 
_Depressants:_
Amytal
Diazepam
Halcion
Klonopin
Librium
Phenobarbital
Seconal
Valium
Xanax
 
_Inhalants:_
Cheese Spray Can
Cleaning Fluid
Correction Fluid
Felt Marker
Fingernail Polish
Gasoline
Glue
Gas Canister
Jet
Laughing Gas
Meth
Paint
Paint Thinner
Spice
Whippits
Whipped Cream Can
 
_Opiates:_
Buprenex/Suboxone
Heroin
Hydrocodone
Methadone
Morphine
Vicodin
 
_Steroids:_
Anadrol
Dianabol
Durabolin
Testosterone
Tramadol
Trenbolone
(If you want to be a memelord, you can add creatine in here.)
 
_Alcohol:_
Absinthe
Ale
Beer
Bourbon
Brandy
Chardonnay
Grog
Mead
Moonshine
Rot Gut
Rum
Scotch
Vodka
Tequila
Whiskey
Wine
 
_Other Drugs:_
Cigarette
Pack of Cigarettes
Carton of Cigarettes
E-Cig
Cigar
Black & Mild
Joint
Marijuana
Nicotine Gum
Chewing Tobacco
Rolling Tobacco
Nicotine Liquid
Synthol
Agent Orange
Rat Poison
Krokodil
DMT
Shrooms
 
_Fictional Drugs:_
Mentats
Jet
Stim Shot
Spice

 
_Personal Hygene Items:_
Aftershave
Cologne
Cotton Pad
Cotton Swab
Deodorant
Disposable Towel
Dry Shampoo
Eye Liner
Facial Tissue
Gel Wipe
Hand Dryer
Hand Sanitizer
Handkerchief
Lip Gloss
Lipstick
Lotion
Makeup
Menstrual Cup
Moisturizer
Pomade
Perfume
Razor
Shampoo
Shampoo- Conditioner
Shaving Cream
Shaving Oil
Shoe Polish
Shower Gel
Soap Bar
Soap Liquid
Talcum Powder
Tampon
Toilet Paper
Tooth Paste
Towel
Wet Wipe

**Containers (Consumables):**

Aluminum Foil
Bag- Canvas
Bag- Cloth
Bag- Garbage
Bag- Hemp
Bag- Messenger
Bag- Messenger, Officer
Bag- Paper
Bag- Plastic
Sack- Burlap
Sack- Canvas
Sack- Hemp
Sack- Plastic
Sack- Potato
Sack- Vacuum
Barrel- Iron, Gasoline
Barrel- Iron, Water
Barrel- Wood, Alcohol
Barrel- Wood, Food
Bottle- Copper
Bottle- Iron
Bottle- Glass
Bottle- Plastic
Bottle- Tin
Box- Cardboard
Box- Glass
Box- Iron
Box- Plastic
Box- Steel
Box- Styrofoam
Box- Tin
Box- Wood
Can- Aluminum
Can- Copper
Can- Plastic
Can- Tin
Canteen
Crate- Iron
Crate- Wood
Cup- Cardboard
Cup- Glass
Cup- Styrofoam
Flask- Glass
Flask- Copper
Flask- Iron
Flask- Steel
Jug- Gallon, Plastic
Jug- Gallon, Steel
Jug- Quart, Plastic
Jug- Quart, Steel
Wrapper- Paper
Wrapper- Plastic

This completes the non-comprehensive list of consumable items. 

**_### Objects, Deployables, Workshops, and More:_**

**Containers (Home Storage):**

_In-Building Home Sites:_
Burrow
Box Fort
Concrete Hovel
Lean-To
Debris Hut
Debris Wall
Hogan
Pit- Empty
Pit- Glass
Pit- Spike
Shack- Metal
Shack- Wood
Shelter- Improvised
Shelter- Tarp
Thatched Hut
Tent
Trench
Guard Post- Medium
Guard Post- Large
 
_Work Stations/Appliances:_
Air Conditioner
Air Purifier
Anvil
Aroma Lamp
Back Boiler
Can/Tin Opener
Clothes Dryer
Clothes Washer
Clothes Washer & Dryer
Computer
Dishwasher
Domestic Robot
Fan Heater
Fermenting Vat
Fireplace
Forced Air System
Forge
Garbage Disposal
Gas Underfloor Heater
Hearth
Humidifier
HVAC
Icebox
Manual Vacuum
Mechanical Clothes Cleaner
Microwave
Oven
Portable Heater
Portable Fan
Radiator
Refrigerator
Refrigerator- Chest
Freezer
Freezer- Chest
Window Refrigerator
Window Freezer
Window Refrigerator- Chest
Window Freezer- Chest
Still
Sump/Pump
Toaster
Water Boiler
Water Purifier
Water Softener
Window Fan
Television
 
_Large Normal Containers:_
Bookcase
Bulletin Board
Cabinet- End
Cabinet- Filing
Cabinet- Tall
Cabinet- Waist
Cabinet- Wall
Counter
Console
Crate
Cupboard
Desk
Display Case- Large
Drawer
Dresser
Hidden Space- Floor (NPCs will not loot these)
Hidden Space- Wall
Hidden Space- Roof
Locker- Standing
Locker- Wall
Racks
Standing Tank
Safe- Box
Safe- Floor
Safe- Wall
Steel Drum
Toolbox- Large
Vending Machine
Weapons Stand
Wooden Keg
 
_Medium Normal Containers:_
Backpack
Bathtub
Box- Cardboard
Box- Metal
Box- Plastic
Cooler
Collander
Collectors Stand
Cubbyhole- Official Docs
Cubbyhole- Painting
Cubbyhole- Rug
Cubbyhole- Writing Board
Display Case- Medium
End Table
Footlocker
Medicine Cabinet
Ottoman
Vending Machine- Medium
WaterBOB
Wetbar
Toolbox
 
_Furniture w/ Small Storage:_
Bed- Mattress
Bed- Makeshift
Bed- Straw
Cistern- Toilet
Shelving
Stuffed Animal
Stuffed Animal- Head
Display Case- Small
 
**_Furniture:_**
_Seating- Single:_
Chair
*Lift Chair
Bean Bag
Chaise Longue
Fauteuil
Ottoman
Recliner
Stool- Bar
Stool- Foot
Fainting Couch
Rocking Chair
 
_Seating- Multi:_
Bench
Couch- Accubita (2 ends)
Couch- Canape (Round Back)
Couch- Davenport (Sofa)
Couch- Divan (Matresslike)
Couch- Loveseat (2 seat)
 
_Sleeping/Lying:_
Bend- Bunk
Bed- Canopy (Curtain)
Bed- Four-Poster
Bed- Murphy (Wall)
Bed- Platform
Bed- Sleigh
Bed- Water
Daybed
Futon
Hammock
Mattress
Sofa Bed
 
_Entertainment:_
Billiard Table
Chess Table
Entertainment Center
Gramophone
Hi Fi Speaker
Jukebox
Pinball Machine
Radiogram
Television/Set
Radio Receiver
Video Game Console

**Home Appliances/Upgrades:**

_Work Stations:_
Work Bench
Brewery
Distillery
Anvil
Quern (Mill/Grinder)
Screw Press (Oil/Honey Mats)
Loom
Tannery/Module
Kitchen/Fire/Module

Modules for Workbench, or perhaps their own separate benches as deemed necessary for balance. These would decrease crafting times for their respective fields, and also have unique features, such as recipes that rely upon them as a non-consumed ingredient, and secondary purposes such as comms to talk to any companion holding a two-way radio:
Research (Unlocks recipes that cannot be learned in books for end game. Things such as teleporters and suspensors can go here for balance.)
Ammunition
Armor
Butchery
Carpentry
Chemistry
Weapons
Electronics
Fabrication
Mechanics
Metalworking
Stoneworking
Woodworking
Surveillance (Can see through posted cameras on owned vehicles if turned on, as well as static cameras in local building grid.)
Comms (Talk to companions)
Scavenging (Send companions to gather, and bring their materials here.)
Robotics (Create modular robot companions to do your bidding and take over the world.)
 
_Appliances & Electrical:_
Air Conditioning
Bread Machine
Coffeemaker
Crockpot
Deep Fryer
Dish Washer
Dryer & Washer
Food Processor
Food Dehydrator
Forge- Electrical
Garbage Disposal/Processor
Hot Plate
Microwave
Oven
Oven- Convection/Portable
Radio
Radio- HAM
Refrigerator
Refrigerator- Chest
Security Camera
Sewing Machine/Station
Vacuum Sealer
Water Heater- Storage
Water Purifier
 
_Power:_
Alternator- Car
Alternator- Motorcycle
Alternator- Truck
Battery- Disposable A
Battery- Disposable AA
Battery- Disposable AAA
Battery- Rechargeable
Battery- Car
Battery- Electric Vehicle
Battery-Motorcycle
Battery- Truck
Generator- Diesel
Generator- Foot Crank
Generator- Gasoline
Generator- Solar
Generator- Thermal
Solar Cells
Solar Panels
 
**_### _Buildings:_  (For settlement building classification purposes)_**
Scrapyard
Mine/Quarry
Shed
Caravan Post
Trade Depot
Kennel Shop
Auto Shop
Trade Shop- General Store
Trade Shop- Scrap Store
Trade Shop- Pawn Shop
Trade Shop- Medical
Trade Shop- Restaurant
Trade Shop- Clothes&Tailor
Trade Shop- Smith
Trade Shop- Mercenary
Trade Shop- Contractors
 
_Misc:_
Bell
Siren
Meet Up Location
 
_Resources:_
Water:
Cistern- 1,500 Gal
Cistern- 5,000 Gal
Cistern- 12,000 Gal
Storage Tank- 50 Gal
Storage Tank- 200 Gal
Underground ST- 50 Gal
Underground ST- 200 Gal
Water Pump
Well
Rain Collector- Bucket Size
Rain Collector- Barrel Size
Rain Collector- Pool Size
 
_Food: (Animals/Livestock)_
Small Pen- 5x5ft
Medium Pen- 15x15 ft
Large Pen- 40x 40 ft
Grazing Pen- 100x100ft+
Feeding Trough-25L/50M/100
Watering Hole/Pond- 400 Gal
Water Dispenser- 50 Gal
Water Barrel- 5 Gal
Food Dispenser: 1L/2M/4S
Incubator
Shelter Area
 
_Food (Plant Matter):_
Burlap Mushroom Planter
Mushroom Growth Kit
Aquaponics Station
Aquaponics Box
Garden Plot- Outdoor
Garden Plot- Indoor/Under
Greenhouse
Greenhouse Box
Growing Pot
Hydroponics Station
Hydroponics Box
Planter Box
Tilled Plot

Again, I hope this is helpful to you and the community at large, and can serve as a source of inspiration, or even what not to do in terms of overbearing level of detail. Or, if preferred, this can be used to provide more granular detail to the game as appropriate. I leave it in your able hands, but will be adding said items to a fork for a pull request at your behest. Thank you for your time.

Very Respectfully,
Eleusius.

**Comment by scarf005** [comment]
hi, thanks for the new item ideas!
however, adding multiple item variants with current system could be taxing to system and item UI unless solution like #3504 are implemented.
It might be a good idea to make a mod first, then gradually introduce PRs for mainlining.

~~_Also we don't use gallons and feets, we use the true freedom unit, SI_~~
